Zend/zend_vm_def.h

```diff
@@ -7114,7 +7114,10 @@ ZEND_VM_HELPER(zend_dispatch_try_catch_finally_helper, ANY, ANY, uint32_t try_ca
 		zend_try_catch_element *try_catch =
 			&EX(func)->op_array.try_catch_array[try_catch_offset];
 
-		if (op_num < try_catch->catch_op) {
+		if (op_num < try_catch->try_op) {
```

Comment:
Not sure if this check makes sense: try_op always increases, so if op_num > try_op once, then it will still be greater if we go to a previous try_op.